### PR TITLE
Include object key as ID

### PR DESCRIPTION
**Pull request by grdddj**
Fixes https://github.com/trezor/definitions/issues/14:
- adds the object key for all the networks and tokens as its `id`
- also updating the tokens lists for the latest state

### CODE REVIEW

**Review by matejcik — APPROVED** [review]
ACK from me, let's wait for approval from @techi602

**Review by techi602 — APPROVED** [review]
Thank You!

**Pull request merged**
